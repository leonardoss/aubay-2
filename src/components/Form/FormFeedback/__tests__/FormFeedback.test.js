import React from 'react';
import FormFeedback from '../FormFeedback';
import renderer from 'react-test-renderer';

it('FormFeedback should renders correctly', () => {
  const tree = renderer.create(<FormFeedback />).toJSON();
  expect(tree).toMatchSnapshot();
});