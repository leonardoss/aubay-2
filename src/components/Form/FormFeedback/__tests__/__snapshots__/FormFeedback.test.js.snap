// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormFeedback should renders correctly 1`] = `
<span
  className="msg-hidden"
/>
`;
